clients/go/cmd/zbctl/main_test.go

```diff
@@ -48,86 +48,104 @@ type integrationTestSuite struct {
+		cmd:        strings.Fields("--insecure deploy testdata/model.bpmn testdata/job_model.bpmn --resourceNames=model.bpmn,job.bpmn"),
 		goldenFile: "testdata/deploy.golden",
 		jsonOutput: true,
 	},
 	{
 		name:       "create instance",
-		setupCmds:  []string{"--insecure deploy testdata/model.bpmn"},
-		cmd:        "--insecure create instance process",
+		setupCmds:  [][]string{strings.Fields("--insecure deploy testdata/model.bpmn")},
+		cmd:        strings.Fields("--insecure create instance process"),
 		goldenFile: "testdata/create_instance.golden",
 		jsonOutput: true,
 	},
 	{
-		name:       "create worker",
-		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess"},
-		cmd:        "create --insecure worker jobType --handler echo",
+		name: "create worker",
+		setupCmds: [][]string{
+			strings.Fields("--insecure deploy testdata/job_model.bpmn"),
+			strings.Fields("--insecure create instance jobProcess"),
+		},
+		cmd:        strings.Fields("create --insecure worker jobType --handler echo"),
 		goldenFile: "testdata/create_worker.golden",
 	},
 	{
 		name:       "empty activate job",
-		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 0",
+		cmd:        strings.Fields("--insecure activate jobs jobType --maxJobsToActivate 0"),
 		goldenFile: "testdata/empty_activate_job.golden",
 		jsonOutput: true,
 	},
 	{
-		name:       "single activate job",
-		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess"},
-		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 1",
+		name: "single activate job",
+		setupCmds: [][]string{
+			strings.Fields("--insecure deploy testdata/job_model.bpmn"),
+			strings.Fields("--insecure create instance jobProcess"),
+		},
+		cmd:        strings.Fields("--insecure activate jobs jobType --maxJobsToActivate 1"),
 		goldenFile: "testdata/single_activate_job.golden",
 		jsonOutput: true,
 	},
 	{
-		name:       "double activate job",
-		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess", "--insecure create instance jobProcess"},
-		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 2",
+		name: "double activate job",
+		setupCmds: [][]string{
+			strings.Fields("--insecure deploy testdata/job_model.bpmn"),
+			strings.Fields("--insecure create instance jobProcess"),
+			strings.Fields("--insecure create instance jobProcess"),
+		},
+		cmd:        strings.Fields("--insecure activate jobs jobType --maxJobsToActivate 2"),
 		goldenFile: "testdata/double_activate_job.golden",
 		jsonOutput: true,
 	},
+	{
+		name:      "send message with a space",
+		setupCmds: [][]string{strings.Fields("--insecure deploy testdata/start_event.bpmn")},
+		// we have to use a "`" symbol to create a command argument with quotes.
+		cmd:        []string{"--insecure", "publish", "message", "`Start Process`", "--correlationKey", "1234"},
```

Comment:
```suggestion
		cmd:        []string{"--insecure", "publish", "message", "Start Process", "--correlationKey", "1234"},
```
This works without the backticks. Can you confirm that it works for you as well?

Reply:
Hey @MiguelPires! Yep, it works, but I think that the test should be more robust (I would like to verify that a process instance will be created after sending a message). WDYT? Or we can leave this test as is (just verify that there will be no error if we use arguments with spaces)?